### DISCUSSION TITLE
Why i Have missing layers ?

### DISCUSSION THREAD

**Discussion by Bersaker** [OrcaSlicer/OrcaSlicer#284] [Q&A] [answered]
I would like to print a part, but I notice that layers are always missing. what can be the reason?

![missing layer](https://user-images.githubusercontent.com/62776321/216830366-fa4290fe-61a1-4c93-9cbe-cfb24becf015.JPG)

**Comment by hliebscher** [comment, marked as answer]
have you the model file ? and print settings ? or the 3mf file?

**Reply by Bersaker** [reply]
https://github.com/kevinakasam/BeltDrivenEnder3/blob/main/V3/Single_Z/1_Transmission_V3.6.stl

I use profile for ender 3v2... if i use bambustudio, all layers filled.

**Comment by hliebscher** [comment]
witch software version? for me with your model is fine
![bug](https://user-images.githubusercontent.com/9274162/216846798-a5ffd03c-00f8-40f1-a1ed-acd1c2e0195b.jpg)

**Reply by Bersaker** [reply]
The latest version 1.4.4

**Reply by hliebscher** [reply]
mac or windows can you change the printer model for testing?

**Reply by hliebscher** [reply]
> The latest version 1.4.4

[1_Transmission_V3.6.3mf.zip](https://github.com/SoftFever/BambuStudio-SoftFever/files/10613901/1_Transmission_V3.6.3mf.zip)

**Comment by erebus04** [comment]
I had the same problem, but it was just a display problem. After updating Windows and the drivers for the graphics card, the problem was resolved.
